### PR TITLE
Add support for output of relperm to .INIT file with SLGOF input.

### PR DESCRIPTION
**Pull request by vkip**

### CODE REVIEW

**Comment by vkip** [comment]
jenkins build this please

**Comment by vkip** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
I think this looks good for the most part, although I'd normally prefer checking `!TableContainer::empty()` instead of `TableContainer::size() > 0`.  That said, there may be one incorrect condition here, so please check carefully where I've posed I direct question.  We also need to understand why so many regression tests fail as a result of this PR.

File: opm/output/eclipse/Tables.cpp [hunk 1]
```diff
@@ -1379,7 +1565,15 @@ namespace { namespace SatFunc {
                 const auto& gasLET = tabMgr.getSgofletTable();
 
                 if (gasLET.empty()) {
-                    return fromSGOF(numRows, tolcrit, tabMgr.getSgofTables());
+                    const auto& sgof = tabMgr.getSgofTables();
+                    if (sgof.size() > 0) {
+                        return fromSGOF(numRows, tolcrit, sgof);
+                    }
+                    const auto& slgof = tabMgr.getSlgofTables();
+                    if (sgof.size() > 0) {
```
[the remark this answers is not in the document]
Comment (vkip) [reply opm/output/eclipse/Tables.cpp:1573]: Thanks, good point! Will update.

**Comment by vkip** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and as I finally understand how you fixed the initial regression test failures I'll merge into master.  We should look into the equivalence predicate for joint saturations at a later date.

**Comment by vkip** [comment]
> Thanks a lot for the updates. This looks good to me now and as I finally understand how you fixed the initial regression test failures I'll merge into master. We should look into the equivalence predicate for joint saturations at a later date.

Thanks! I think it would be a good idea to use a non-zero default tolerance to mergeTables and update the test results, as there are currently lots of nearly-duplicate saturation values in the .INIT output. I'll create a PR and start looking at the test results..

**Comment by bska** [comment]
> > Thanks a lot for the updates. This looks good to me now and as I finally understand how you fixed the initial regression test failures I'll merge into master. We should look into the equivalence predicate for joint saturations at a later date.
> 
> Thanks! I think it would be a good idea to use a non-zero default tolerance to mergeTables and update the test results, as there are currently lots of nearly-duplicate saturation values in the .INIT output. I'll create a PR and start looking at the test results..

Sounds good.  For this purpose, a threshold of $10^{-6}$ or even $10^{-5}$ might be reasonable.

**Comment by vkip** [comment]
> Sounds good. For this purpose, a threshold of 10 − 6 or even 10 − 5 might be reasonable.

Thanks, trying with 1e-6: https://github.com/OPM/opm-common/pull/4313

**Pull request merged**